shell/platform/android/io/flutter/embedding/engine/loader/ApplicationInfoLoader.java

```diff
@@ -27,6 +27,8 @@
   public static final String PUBLIC_FLUTTER_ASSETS_DIR_KEY =
       FlutterLoader.class.getName() + '.' + FlutterLoader.FLUTTER_ASSETS_DIR_KEY;
   public static final String NETWORK_POLICY_METADATA_KEY = "io.flutter.network-policy";
+  public static final String PUBLIC_AUTOMATICALLY_REGISTER_PLUGINS_METADATA_KEY =
+      FlutterLoader.class.getName() + '.' + FlutterLoader.AUTOMATICALLY_REGISTER_PLUGINS_KEY;
```

Comment:
`"io.flutter.automatic-plugin-registration"`. hyphens to follow the current convention.

Reply:
Done.